storio-common-annotations-processor/src/main/java/com/pushtorefresh/storio/common/annotations/processor/introspection/StorIOTypeMeta.java

```diff
@@ -17,19 +20,33 @@
     @NotNull
     public final TypeAnnotation storIOType;
 
+    public boolean needCreator;
+
+    @Nullable
+    public ExecutableElement creator;
+
     /**
      * Yep, this is MODIFIABLE Map, please use it carefully
      */
     @NotNull
-    public final Map<String, ColumnMeta> columns = new HashMap<String, ColumnMeta>();
+    public final Map<String, ColumnMeta> columns = new LinkedHashMap<String, ColumnMeta>();
```

Comment:
Yeah, it's used for ordering, since constructor/factory method parameters depend on it.